### PR TITLE
Updated readme

### LINKED ISSUE #2: Update README for how to add to the handbook
Coming soon!

### PR DESCRIPTION
**Pull request by margarethaley**
This pr updates the readme.md file for the community handbook.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,36 @@
...
 
-## License
+## How to Contribute
 
-This template ensures new repos are created compliant with [ADR 0001](https://www.operate-first.cloud/blueprints/blueprint/docs/adr/0001-use-gpl3-as-license.md) and use GNU GPL v3 license.
+We welcome the Operate First community to contribute to the handbook!
+There are multiple ways to contribute:
+1. Create an issue in our [GitHub repository](https://github.com/operate-first/community-handbook)
+1. Create a pull request (PR) directly against the repository
+1. Label an issue or PR from another part of the project to identify as part of the Community Handbook
```
Comment (oindrillac) [inline README.md:15]: should the issue be labeled /area community, if so should we specify it here?

Comment (margarethaley) [reply README.md:15]: @quaid and I discussed which labels to apply and chose area/contributor and kind/handbook to signify issues and prs for the Community Handbook. In the future, one of us will write out explicit directions for how to link other parts of the project to the handbook, but we just haven't gotten to it yet!

**Pull request merged**